models/intel/text-spotting-0003/description/text-spotting-0003.md

```diff
@@ -0,0 +1,102 @@
+# text-spotting-0003 (composite)
+
+## Use Case and High-Level Description
+
+This is a text spotting composite model that simultaneously detects and
+recognizes text. The model detects symbol sequences separated by space and performs
+recognition without a dictionary. The model is built on top of the Mask-RCNN
+framework with additional attention-based text recognition head.
+
+Symbols set is alphanumeric: `0123456789abcdefghijklmnopqrstuvwxyz`.
+
+## Example
+
+![](./text-spotting-0003.png)
+
+## Composite model specification
+
+| Metric                                        | Value     |
+|-----------------------------------------------|-----------|
+| Word spotting hmean ICDAR2015, without a dictionary | TBD |
+| Source framework                              | PyTorch\* |
+
+*Hmean Word spotting* is defined and measured according to the
+[Incidental Scene Text (ICDAR2015) challenge](https://rrc.cvc.uab.es/?ch=4&com=introduction).
+
+## Detector model specification
+
+The text-spotting-0003-detector model is a Mask-RCNN-based text detector with ResNet50 backbone and additional text features output.
+
+| Metric                                        | Value     |
+|-----------------------------------------------|-----------|
+| GFlops                                        | TBD       |
+| MParams                                       | TBD       |
+
+
+### Performance
```

Comment:
we are removing empty sections from model documentation (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)), please remove empty section here as well